llvm/include/llvm/CodeGen/ValueTypes.td

```diff
@@ -357,6 +362,11 @@ def amdgpuBufferStridedPointer : ValueType<192, 252>;
 
 def aarch64mfp8 : ValueType<8,  253>;  // 8-bit value in FPR (AArch64)
 
+def c64
+    : VTCapability<64, !add(aarch64mfp8.Value, 1)>; // 64-bit capability value
```

Comment:
Looks like everything else in here is explicitly numbered rather than adding over the previous.

Reply:
Done.

Reply:
Yeah I believe I changed it to an add downstream to make it easier to resolve merge conflicts. With it being upstream that will no longer be an issue.